File: JAVA/Tables.java

import java.util.Scanner;

public class Tables {
    public static void main(String[] args){
        Scanner lol = new Scanner(System.in);

        System.out.print("ENTER ANY NUMBER: ");
        int ae = lol.nextInt();

        for(int a = 1; a <= 10; a++){
            System.out.print(ae + " x " + a + " = " + (ae * a)+"\n");
        }
        lol.close();
    }
}

File: C/tables.c
#include <stdio.h>

int main() {
    int lol;

    printf("Enter number: ");
    scanf("%d", &lol);

    int hehe;
    printf("how many times you want to multiply: ");
    scanf("%d", &hehe);

    for (int a = 1; a <= hehe; a++) {
        printf("%d x %d = %d\n", lol, a, lol * a);
    }

    return 0;
}

